tests/Unit/AggregateAttributeLoaderTest.php

```diff
@@ -0,0 +1,35 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Patchlevel\EventSourcingBundle\Tests\Unit;
+
+use Patchlevel\EventSourcing\Aggregate\AggregateChanged;
+use Patchlevel\EventSourcing\Aggregate\AggregateRoot;
+use Patchlevel\EventSourcingBundle\Attribute\Aggregate;
+use Patchlevel\EventSourcingBundle\Loader\AggregateAttributesLoader;
+use PHPUnit\Framework\TestCase;
+use InvalidArgumentException;
+
+class AggregateAttributeLoaderTest extends TestCase
+{
+    public function testInvalidArgumentException()
```

Comment:
```suggestion
    public function testDuplicateAggregateName()
```

And, if we use a generic exception, we should check if the exception message is right.